metricshub-oscommand-extension/src/main/java/org/metricshub/extension/oscommand/CommandLineSourceProcessor.java

```diff
@@ -96,12 +96,7 @@ public SourceTable process(
 			return SourceTable
 				.builder()
 				.rawData(selectedColumnsLines.stream().collect(Collectors.joining(NEW_LINE)))
-				.table(
-					selectedColumnsLines
-						.stream()
-						.map(line -> Stream.of(line.split(TABLE_SEP)).collect(Collectors.toList()))
-						.collect(Collectors.toList())
-				)
+				.table(selectedColumnsLines.stream().map(line -> Stream.of(line.split(TABLE_SEP)).toList()).toList())

```

Comment:
This introduces a regression. Please keep `Collectors.toList()` as is, or use `Collectors.toCollection(...)` to make the intent explicit.